FV3/atmos_model.F90

```diff
@@ -296,6 +297,11 @@ subroutine update_atmos_radiation_physics (Atmos)
     call atmos_phys_driver_statein (IPD_data, Atm_block, flip_vc)
     call mpp_clock_end(getClock)
 
+    ! Get prescribed sea surface temperatures and sea ice (if using)
+    call mpp_clock_begin(overrideClock)
+    call sfc_data_override(Atmos%Time, IPD_data, Atm_block, IPD_Control)
```

Comment:
Indeed it will raise an error like this if the file specified in the `data_table` cannot be found:
```
FATAL from PE     0: MPP_OPEN:INPUT/sst.nc does not exist.
```
or an error like this if the `data_table` is missing or misconfigured:
```
FATAL from PE     3:  SST dataset not specified in data_table.
```